packages/components/src/components/tab-panel/tab-panel.ts

```diff
@@ -46,16 +43,28 @@ export default class SdTabPanel extends SolidElement {
   }
 
   render() {
-    return html`
-      <slot
-        part="base"
-        class=${classMap({
-      'tab-panel': true,
-      'tab-panel--active': this.active
-    })}
-      ></slot>
-    `;
+    return html` <slot part="base" class=${cx('tab-panel block', this.active && 'tab-panel--active')}></slot> `;
   }
+
+  static styles = [
+    SolidElement.styles,
+    componentStyles,
+    css`
+      :host {
+        --padding: 0;
+
+        @apply hidden;
+      }
+
+      :host([active]) {
+        @apply block;
+      }
+
+      .tab-panel {
+        padding: var(--padding);
```

Comment:
is the tailwind class `p-0` not working?

Reply:
Here I am only setting the CSS variable we wanted to expose so it wouldn't always be p-0. I wasn't sure if there is a way to set it as a variable in tailwind.